Controller/FOSRestController.php

```diff
@@ -17,6 +17,8 @@
 /**
  * Controllers using the View functionality of FOSRestBundle.
  *
+ * @deprecated when used with Symfony 4.2, use {@see AbstractFOSRestController} instead.
```

Comment:
I think we should just deprecate the class:

```php
/**
 * @deprecated since FOSRestBundle 2.5, use {@see AbstractFOSRestController} instead
 */
```